packages/counterfactual_nft/migrations/2_deploy_contracts.js

```diff
@@ -0,0 +1,26 @@
+const NftContract = artifacts.require("./CounterfactualNFT.sol");
+const NFTFactory = artifacts.require("./NFTFactory.sol");
+
+module.exports = async (deployer, network, addresses) => {
+  console.log("Deploying to " + network);
+
+  const loopringExchangeAddress = "0x0BABA1Ad5bE3a5C0a66E7ac838a129Bf948f1eA4";
+
+  let owner;
+  if (network === 'rinkeby') {
+    owner = addresses[0];
+  } else if (network === 'development') {
+    owner = addresses[0];
+  } else {
+    owner = addresses[0];
+  }
```

Comment:
`owner` is always the same?

Reply:
Ah good point. Left it there to make it easier to add different addresses again on different networks but that was for the deployment of different contracts and it's not even useful here. Removed, thanks!